crates/ibc/Cargo.toml

```diff
@@ -75,7 +75,7 @@ scale-info = { version = "2.1.2", default-features = false, features = ["derive"
 borsh = {version = "0.9.0", default-features = false, optional = true }
 parking_lot = { version = "0.12.1", default-features = false, optional = true }
 
-ibc-derive = "0.2.0"
+ibc-derive = { version ="0.2.0", path = "../ibc-derive" }
```

Comment:
Note: I had to reincorporate the `path` because otherwise whenever we make changes to `ibc-derive`, `ibc` doesn't see the changes, and it fails to compile

Reply:
Therefore, good to add a `[ibc-drive]` tag for unclog. Like how this handled e.g. in [`tendermint-rs`](https://github.com/informalsystems/tendermint-rs/blob/main/CHANGELOG.md#breaking-changes)

Reply:
Was thinking of adding this to the `contributing.md`. I see we already have it [here](https://github.com/cosmos/ibc-rs/blob/c4d69986c1fb075b00512f39be84719477f11f64/CONTRIBUTING.md#preview-unreleased-changes)